compiler/IREmitter/Payload/codegen-payload.c

```diff
@@ -147,9 +147,9 @@ SORBET_ALIVE(VALUE, sorbet_magic_mergeHashHelper, (VALUE, VALUE));
 SORBET_ALIVE(VALUE, sorbet_vm_getivar, (VALUE obj, ID id, struct iseq_inline_iv_cache_entry *cache));
 SORBET_ALIVE(void, sorbet_vm_setivar, (VALUE obj, ID id, VALUE val, struct iseq_inline_iv_cache_entry *cache));
 
-SORBET_ALIVE(int, sorbet_initializeTag, (struct rb_vm_tag * tag));
+SORBET_ALIVE(enum ruby_tag_type, sorbet_initializeTag, (struct rb_vm_tag * tag));
 SORBET_ALIVE(VALUE, sorbet_processThrowReturnSetJmp,
-             (int setjmp_retval, rb_control_frame_t *cfp, struct rb_vm_tag *tag));
+             (enum ruby_tag_type state, rb_control_frame_t *cfp, struct rb_vm_tag *tag));
```

Comment:
Same for this function. If it's defined in the codegen payload, I don't think we need to use `sorbet_alive`.

Reply:
I've used `KEEP_ALIVE` for the two functions I modified here and for `sorbet_teardownTagForThrowReturn`.